ReactWindows/ReactNative.Shared/UIManager/BorderedViewParentManager.cs

```diff
@@ -88,11 +111,15 @@ public void SetBackgroundColor(Border view, uint color)
         /// <param name="view">The view panel.</param>
         /// <param name="color">The color hex code.</param>
         [ReactProp("borderColor", CustomType = "Color")]
-        public void SetBorderColor(Border view, uint? color)
+        public void SetBorderColor(TFrameworkElement view, uint? color)
         {
-            view.BorderBrush = color.HasValue
-                ? new SolidColorBrush(ColorHelpers.Parse(color.Value))
-                : s_defaultBorderBrush;
+            if (HasBorder(view) || color.HasValue && !ColorHelpers.IsTransparent(color.Value))
```

Comment:
Fair enough, why not just eliminate the optimization?

---
In reply to: [127868591](https://github.com/rozele/react-native-windows/pull/1#discussion_r127868591) [](ancestors = 127868591)